examples/event-desc-descType.xml

```diff
@@ -0,0 +1,26 @@
+...
+  <body>
+    <div begin="10s" end="13s">
+      <metadata>
+        <ttm:desc daptm:descType="scene">Scene 1</ttm:desc>
```

Comment:
Probably worth putting two different `ttm:desc`s in with different types to make it clear why we're bothing, for example maybe something like:

```suggestion
        <ttm:desc daptm:descType="scene">Scene 1</ttm:desc>
        <ttm:desc daptm:descType="plotSignificance">High</ttm:desc>
```

At https://w3c.github.io/adpt/#bbc-example-section the two values used are for "pronunciation note" and "shot description" - if they sound like better proposals for the example I'm happy to use those instead.

Also they're both `ttm:desc` elements that are children of the `<p>` element, i.e. no need to make them children of a `<metadata>` element - this is clearly allowed in TTML2, by the way, so may be worth doing as well, as a simplification?